src/DependencyInjection/KnpMenuExtension.php

```diff
@@ -75,4 +66,55 @@ public function prepend(ContainerBuilder $container): void
 
         $container->prependExtensionConfig('twig', ['paths' => [$path]]);
     }
+
+    /**
+     * @param $providers
+     * @param ContainerBuilder $container
+     */
+    private function GetDefinitionsForEnabledProviders($providers, ContainerBuilder $container): void
+    {
+        foreach ($providers as $builder => $enabled) {
+            $this->GetDefinitionForEnabledProvider($enabled, $container, $builder);
+        }
+    }
+
+    /**
+     * @param $enabled
+     * @param ContainerBuilder $container
+     * @param $builder
+     */
+    private function getDefinitionForEnabledProvider($enabled, ContainerBuilder $container, $builder): void
+    {
+        if (!$enabled) return ;
+
+        $container->getDefinition(\sprintf('knp_menu.menu_provider.%s', $builder))->addTag('knp_menu.provider');
+    }
+
+    /**
+     * @param array $config
+     * @param XmlFileLoader $loader
+     * @param ContainerBuilder $container
+     * @return array
+     * @throws \Exception
+     */
+    private function loadTwigConfiguration(array $config, XmlFileLoader $loader, ContainerBuilder $container): array
+    {
+        if (isset($config['twig'])) {
+            $loader->load('twig.xml');
+            $container->setParameter('knp_menu.renderer.twig.template', $config['twig']['template']);
+        }
+        return $config;
```

Comment:
Here you miss an empty line before returning. I guess that you missed running the cs-fixer on your code.